inhouse/command_handlers/match.py

```diff
@@ -232,7 +240,9 @@ def create_missing_players(self):
         existing_player_ids = [existing_player[0] for existing_player in cur.fetchall()]
 
         for player in all_players:
-            if not player.id in existing_player_ids:
+            if player.id == -1:
```

Comment:
nit: small comment over this indicating that test players get an id of -1 might be useful. Not a biggie though, feel free to ignore.

Reply:
Updated